Specs/Scene/ImplicitSubtreeSpec.js

```diff
@@ -640,33 +658,37 @@ describe("Scene/ImplicitSubtree", function () {
   });
 
   describe("3DTILES_multiple_contents", function () {
-    var multipleContentsQuadtree = new ImplicitTileset(tilesetResource, {
```

Comment:
Note to self: check through these unit tests carefully. I recall having a bad merge a couple days ago.